emperor/support_files/js/controller.js

```diff
@@ -162,19 +162,24 @@ define([
 
     this.renderer.setSize(plotWidth, this.height);
 
-    if (this.colorController !== undefined){
-      // resize the grid according to the size of the container, since we are
-      // inside the tabs we have to account for that lost space, hence the
-      var tabWidth = this.$plotMenu.width() * this.GRID_SCALE,
-      tabHeight = this.$plotMenu.height() * this.GRID_SCALE;
+    // resize the grid according to the size of the container, since we are
+    // inside the tabs we have to account for that lost space, hence the
```

Comment:
I just noticed this comment is a bit off, though it was like this before, would you mind deleting the "hence the" at the end of this line?